src/postgres/commands/registerPostgresCommands.ts

```diff
@@ -23,11 +23,12 @@ import { executePostgresQuery } from "./executePostgresQuery";
 import { openPostgresFunction } from "./openPostgresFunction";
 
 export const connectedPostgresKey: string = 'ms-azuretools.vscode-azuredatabases.connectedPostgresDB';
-const postgresLanguageId: string = 'sql';
+export const postgresFileExtension: string = '.sql';
```

Comment:
nit: `sqlFileExtension`

Makes it more clear to me that `postgresBaseFileName` is what you actually use to identify this as postgres instead of just sql